go/name/name.go

```diff
@@ -41,6 +41,9 @@ func ParseProject(name string) (string, error) {
 
 // ParseNote parses the project ID and note ID from a note resource name.
 func ParseNote(name string) (string, string, error) {
+	if name == "" {
```

Comment:
As mentioned below, I think we can just check if o.NoteName is empty, I don't think we want to change the parser.

Reply:
done